src/utils/format.ts

```diff
@@ -98,3 +101,113 @@ export const formatTimeToFromBatch = (
   returnType: 'TIME' | 'BATCH' = 'TIME',
   batchLength = 5,
 ): number => (returnType === 'TIME' ? Number(value) * batchLength : Number(value) / batchLength)
+
+// TODO: move
+// Locale is an empty array because we want it to use user's locale
+const ltFractionFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 18 })
```

Comment:
Are we sure we want to go this road? multiple formatters?

The formatters we have didn't use Intl probably for the issue with big numbers. right?

In my mind it was reusing our other formatter, so we don't have to solve the issues again:
```js
formatSmart (...same arguments as formatAmount) {
   const decimals = getSmartDecimals(parameters)
   return formatAmount({...parameters, decimals })
}
```

Maybe I'm oversimplifying and `getSmartDecimals` should return an optional sufix like `M` or `K`, but I find it simpler

Reply:
EDIT: What, I'm confused. Now I see there's 2 methods: `formatSmart` and `formatAmountForDisplay`. I thought `formatSmart` would be exactly what we use for displaying the amount